nncf/dynamic_graph/transformations/commands.py

```diff
@@ -0,0 +1,62 @@
+from typing import Callable
+
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.transformations.commands import TransformationCommand
+from nncf.common.graph.transformations.commands import TransformationPriority
+from nncf.common.graph.transformations.commands import TransformationType
+from nncf.dynamic_graph.graph import InputAgnosticOperationExecutionContext
+
+
+class PTInsertionPoint(TargetPoint):
```

Comment:
There is no PyTorch specific call/class in this file so I wonder why everything is called `PT...`?

Reply:
The InputAgnosticOperationExecutionContext right now is a PT-specific entity, and other entities in this file are highly dependent on this at the moment.

Reply:
@vshampor, do you mean that it will be revised during one of the future steps of integration?

Reply:
Sure, we are moving towards a state where all framework-specific entities are collocated in framework-specific folders, but we are doing so gradually.